discord/application.go

```diff
@@ -60,7 +60,8 @@ type Application struct {
 type ApplicationFlags uint32
 
 const (
-	AppFlagGatewayPresence ApplicationFlags = 1 << (iota + 12)
+	AppFlagAutoModerationRuleCreateBadge ApplicationFlags = 1 << 6
```

Comment:
You'll have to move `AppFlagAutoModerationRuleCreateBadge` outside this block. AFAIK, this line being here will bump `iota` up 1, so all the constants after will be off. You could check this; I'm not too sure about it.